### DISCUSSION TITLE
Add support for `startAfter` and `endBefore` Realtime Database filters

### DISCUSSION THREAD

**Discussion by jeromegamez** [kreait/firebase-php#593] [Ideas]
This has been added recently (https://youtu.be/4TKuKKSy5k4)

https://firebase.google.com/docs/database/web/lists-of-data#filtering_data

**Comment by abhlabs** [comment]
There's a note in the realtime database documentation that startAfter and endBefore have no effect:

> The startAfter query filter has been added to the Firebase JS SDK on 2021-02-11. This PHP SDK implements this in the same way as the other filters, but startAfter does not seem to have an effect when used with the Firebase REST API. If you happen to know why or you tried it and it does indeed work, please let me know via the SDK’s git repo.

I'm just wondering if there's any update on this topic.  I can confirm that the same issue exists when you use orderByKey() (assuming I haven't missed a bug in my test code).
